### ISSUE TITLE
Deleting books present in Book History deletes current book from Book History as well

### ISSUE THREAD

**Issue by GoogleCodeExporter** [natowi/prs-plus#178]
```
Reported by Terisa de Morgan, confirmed by me:

What steps will reproduce the problem?
1. Open a book => It goes to Book History
2. Open second book => Goes to Book History
3. Delete a book from the book list
4. Go Home Menu => The history lengh has decreased one item
5. Go Into Book History => The current book disappear and the length decrease 
another item
6. Open the current book (it was the first in the book history)
7. Go to Book History => The current book doesn't appear

In other words: if there is a current book, and you delete other books that are 
also present in Book History, the current book is erroneously deleted from Book 
History as well.
```

Original issue reported on code.google.com by `quisvir` on 24 Sep 2011 at 11:49
